bin/kafka-server-start.sh

```diff
@@ -50,4 +50,4 @@ case $COMMAND in
     ;;
 esac
 
-exec $base_dir/kafka-run-class.sh $EXTRA_ARGS io.confluent.support.metrics.SupportedKafka "$@"
+exec $base_dir/kafka-run-class.sh $EXTRA_ARGS kafka.Kafka "$@"
```

Comment:
we should also update `/bin/kafka-server-stop.sh`?

Reply:
Because we still have the SupportedKafka class (even though deprecated), the stop script has to understand it still.